win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1163,21 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    TRACE("Size Image %d, Size Header %d, ResSize %d\n",
+        pbmiCopy->bmiHeader.biSizeImage, pbmiCopy->bmiHeader.biSize, ResSize);
+
+    /* Test if this is a GCC windres.exe compiled 16 or 32 bpp bitmap using
+     * BI_BITFIELDS and if so, then a mistake causes it not to include
+     * the bytes for the bitfields. So, we have to substract out the
+     * size of the bitfields previously included from bitmap_info_size.*/
```

Comment:
```suggestion
     * size of the bitfields previously included from bitmap_info_size. */
```

Reply:
Yet another good example why // commenting is generally vastly superior. As it avoids potential for such and similar malformatting.

Reply:
Fixed. Thanks.